Tests/Service/Request/RequestFactoryTest.php

```diff
@@ -166,12 +166,12 @@ public function testBuildFlow()
     public function testBuildFlowWithQueryParams()
     {
         $baseUrl = 'baseUrl';
-        $routeString = '/routeString?param={param}';
+        $routeString = '/routeString?query-param={queryParam}';
         $originParamValue = 'value with whitespaces';
         $requestMethod = 'GET';
         $requestBody = '{requestBody:requestBody}';
```

Comment:
I would suggest to either remove $requestBody or switch to POST request here to avoid confusion. But it's minor in such context

Reply:
ok, thanks, good point
it doesn't matter, but I'll remove the request body